py/h2o_wave/server.py

```diff
@@ -311,10 +315,16 @@ async def _receive(self, req: Request):
         access_token = req.headers.get('Wave-Access-Token')
         refresh_token = req.headers.get('Wave-Refresh-Token')
         session_id = req.headers.get('Wave-Session-ID')
+
+        body = await req.json()
+        forwarded_headers = body.get('headers', None)
+        if forwarded_headers:
+            # TODO: Think about how to clean this up.
+            self._forwarded_headers[client_id] = forwarded_headers
```

Comment:
We send headers as CSV. What if split them here to have them in more comfortable form.

Reply:
Hm.. not sure I follow, maybe outdated as well since headers are no longer sent as headers, but rather as a JSON payload which is then unmarshalled into python dict, which is the most comfortable IMO. But happy to discuss further if I am missing something.

Reply:
outdated. I'll take another look at the whole PR.